### ISSUE TITLE
[Feature Request] - Hide Suggestion button on app page based on app.json

### ISSUE THREAD

**Issue by martijnpoppen** [athombv/apps-sdk-issues#379]
Please add a option in the app.json (like donations) to hide the suggestions button on the app page. This is a feature which should be toggle-able for developers so they can decide where they want to be contacted

**Comment by WeeJeWel** [comment]
We have a few competing ideas for this:

1. Hide the 'Suggest' button when a developer specifies this (either by app.json or toggle in the Developer Tools)
2. Instead of a form in the 'Suggest' popup, redirect the user to the Homey Community Forum Topic (when specified in app.json)
3. Show a 'Reply' button + form in the Homey Developer Tools which will send an e-mail to the suggester.

My preference would be #3 because it's by far the most user-friendly (albeit not the best developer-friendly) solution.

What are your thoughts?

**Comment by OlivierZal** [comment]
My preference would be for #2 or #1 (in this order), or even a combination of both or the three options – letting the developer choose which option he wants (from either `app.json` or dev tools UI).

**Comment by martijnpoppen** [comment]
I think all 3 options are fine. However I understood from Herman Meerlo that option 3 would be hard as the dev will be sending the mail from a noreply email address.

**Comment by WeeJeWel** [comment]
> I think all 3 options are fine. However I understood from Herman Meerlo that option 3 would be hard as the dev will be sending the mail from a noreply email address.

Yes, but the Reply-To field will be the developer's e-mail address.
